### DISCUSSION TITLE
How does plugins in shared configs work?

### DISCUSSION THREAD

**Discussion by simpel** [prettier/prettier#16451] [Q&A]
Hi!

We have a project that utilizes a shared config we've created. In that shared config we use plugins, for example the tailwind plugin. But when using it in a project the plugins does not work. Do we need to install all the plugins locally in each project that uses the shared config?

The goal is to have just this `"prettier": "@mycompany/prettier-config"` in our package and that config takes care of everything, incl. plugins....

**Comment by kachkaev** [comment]
That's possible, yes. Example (with plugins):
- https://github.com/kachkaev/website/blob/bc60c18ef7a133b3df994ebfd87d9e25f396ae02/.prettierrc.json
- https://github.com/kachkaev/routine-npm-packages/blob/dacaad03f4950a825b574cf5aeef3217e6b3f677/packages/prettier-config/main.js

I believe you can use `prettier` field in `package.json` instead of `.prettierrc.json` too.

Docs: https://prettier.io/docs/en/configuration.html#sharing-configurations
